notes/src/components/deleteNoteModul.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import axios from 'axios';
+
+class DeleteNoteModul extends React.Component {
+
+    deleteNote = () => {
+        axios.delete(`https://fe-notes.herokuapp.com/note/delete/${this.props.ID}`)
+        .then(response => {
+            this.props.toggleHidden();
+            this.props.history.push("/");
+            this.props.refresh();
+        })
+        .catch(error => console.log(error))
+    }
+
+    render(){
+        return(
+            <div className={`modul-bg ${this.props.hidden}`}>
```

Comment:
your modal doesn't quite work, it pushes the note content down below it if you scroll down, rather than covering it. You would need to use positioning in your css to get this working properly